### PR TITLE
Use inline code blocks for buttons and menu texts in `Performance_troubleshooting`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
According to the [ASC § Keyboard keys](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#keyboard-keys) and [ASC § Button and menu text](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#button-and-menu-text).

Unfortunately, I didn't find all the button texts in the Chinese file, so I'm unsure if there are more texts that need to be highlighted.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by TicClick** [comment]
> Unfortunately, I didn't find all the button texts in the Chinese file, so I'm unsure if there are more texts that need to be highlighted.

I think there might be, and it's just translated

File: wiki/Performance_troubleshooting/zh.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ osu! 会尽最大努力以合理的默认设置运行，并在你做某些会导
 谱面音频倒带或故障。基本上在游戏卡顿时可以在音频中听到。当报告或诊断此问题时，以下方法可能会提供帮助：
 
 - 尝试是否所有的音频设备都存在这个问题。如果你在使用 USB 耳机，请尝试连接电脑的模拟输出接口。
-- 检查在游戏设置中打开“Audio compatibility mode”（音频兼容性模式）是否有帮助。
+- 检查在游戏设置中打开`Audio compatibility mode`（音频兼容性模式）是否有帮助。
```
Comment (Kotoki1337) [inline wiki/Performance_troubleshooting/zh.md:23]:
```suggestion
- 检查在游戏设置中打开 `Audio compatibility mode`（音频兼容性模式）是否有帮助。
```

**Review by Kotoki1337 — CHANGES_REQUESTED** [review]
not miss a lot in the Chinese file, as you can see some of the button texts have been highlighted before. according to the changes you made in English, you also need to highlight all of the `无限制` in the text. They are [L10](https://github.com/ppy/osu-wiki/blob/132b064b3cea758c55fc9d5a8c12169132a35b7f/wiki/Performance_troubleshooting/zh.md?plain=1#L10), [L83](https://github.com/ppy/osu-wiki/blob/132b064b3cea758c55fc9d5a8c12169132a35b7f/wiki/Performance_troubleshooting/zh.md?plain=1#L83), [L95](https://github.com/ppy/osu-wiki/blob/132b064b3cea758c55fc9d5a8c12169132a35b7f/wiki/Performance_troubleshooting/zh.md?plain=1#L95). just using `Ctrl + F` things, you should be able deal them.

**Comment by The-Last-Cookie** [comment]
Is it okay like this or are spaces required for the highlights added in [`eccca5c`](https://github.com/ppy/osu-wiki/pull/7444/commits/eccca5c025235f45da14564f1a7c4264502fc46f) too?

**Comment by Kotoki1337** [comment]
> Is it okay like this or are spaces required for the highlights added in [`eccca5c`](https://github.com/ppy/osu-wiki/pull/7444/commits/eccca5c025235f45da14564f1a7c4264502fc46f) too?

it's not really require, if it's me i will add. currently we dont have explicit criteria for this, usually needed between the Chinese and English

**Review by TicClick — APPROVED** [review]

**Pull request merged**